extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -96,9 +118,9 @@ message PluginConfig {
   // metrics are enabled).
   bool disable_http_size_metrics = 8;
 
-  // Optional. Allows enabling log compression for stackdriver access logs.
-  google.protobuf.BoolValue enable_log_compression = 9;
+  // Optional. Controls whether to export audit log.
```

Comment:
This looks like a breaking change with field re assignment.
Was the original config available in 1.7 ?

Reply:
Yes, the original config is there in 1.7.. Can mark the old one's as deprecated and then remove next cycle?

Reply:
Actually removed it.. thinking more, we need better organization with Audit logs.. can do in 1.9 rather